package/tests/test_api/test_graphql/test_queries.py

```diff
@@ -73,46 +73,98 @@ def test_run_tracking_data_query(
             json={
                 "query": f"""{{runTrackingData
                 (runIds:["{example_run_id}"])
-                {{datasetName, datasetType, data}}}}"""
+                {{groupedDatasetType, datasets{{ datasetName, datasetType, data}}}}}}"""
```

Comment:
Yes, i also think we need to rethink this. Maybe we do groupDataSetType, TrackingDataSetID ? 
and in TrackingDataSet we also have an id and link it that way instead of nesting?